app/code/Magento/Backend/Block/Widget/Tabs.php

```diff
@@ -235,32 +235,88 @@ protected function _setActiveTab($tabId)
      */
     protected function _beforeToHtml()
     {
+        $this->_tabs = $this->reorderTabs();
+        
+        if ($this->_activeTab === null) {
+            /** @var TabInterface $tab */
+            foreach ($this->_tabs as $tab) {
```

Comment:
Hi @tiagosampaio. You can select first array tab uses reset() and current() methods, please use theirs.